ts/packages/lib/src/formatters/ConfigFormat.ts

```diff
@@ -1,5 +1,6 @@
 export type ConfigFormat =
   | 'JSON'
+  | 'CompactJSON'
```

Comment:
Im still not sure regarding the naming, I will come up with a pool and we well let everyone vote on this 🙏🏽